taxonomies/categories.txt

```diff
@@ -32296,9 +32296,18 @@ nl:Drinkyoghurts
 
 <en:Drinkable yogurts
 fr:Yaourts à boire goût fraise, Yaourts à boire fraise, Yaourts à boire saveur fraise
+en:Dairy drink with sugar and fruits, Fermented milk with sugar and fruits, Yogurt with sugar and fruits
```

Comment:
You can add ciqual properties to multiple categories, but then you should not add the en: and fr: entries of the other category.

Reply:
OK, thanks for the heads up, that's exactly what I was wondering... I will fix it in my next commit then.